news-bundle/src/Resources/contao/modules/ModuleNews.php

```diff
@@ -255,11 +255,22 @@ protected function parseArticles($objArticles, $blnAddArchive=false)
 		$count = 0;
 		$arrArticles = array();
 
-		while ($objArticles->next())
+		// Performance: Preload all images in one query so they are loaded into the model registry before actually rendering them
```

Comment:
```suggestion
		// Preload all images in one query so they are loaded into the model registry before actually rendering them
```

I don't think we have to stress that this is a performance feature. The actual comment makes it pretty clear IMHO. But if you want to keep it, that is fine with me as well. 😄